GiTracker/ViewModels/LoginPageViewModel.cs

```diff
@@ -1,14 +1,44 @@
 ﻿using GiTracker.Services.DataLoader;
+using GiTracker.Services.Login;
 using GiTracker.Services.Progress;
 using Prism.Navigation;
+using Xamarin.Forms;
 
 namespace GiTracker.ViewModels
 {
     internal class LoginPageViewModel : BaseViewModel
     {
-        public LoginPageViewModel(ILoader loader, IProgressService progressService, INavigationService navigationService)
+        private readonly ILoginService _loginService;
+        private string _login;
+        private Command _loginCommand;
+        private string _password;
+
+        public LoginPageViewModel(ILoader loader,
+            IProgressService progressService,
+            INavigationService navigationService,
+            ILoginService loginService)
             : base(loader, progressService, navigationService)
         {
+            _loginService = loginService;
+        }
+
+        public string Login
+        {
+            get { return _login; }
+            set { SetProperty(ref _login, value); }
+        }
+
+        public string Password
+        {
+            get { return _password; }
+            set { SetProperty(ref _password, value); }
+        }
+
+        public Command LoginCommand => _loginCommand ?? (_loginCommand = new Command(DoLogin));
```

Comment:
I think there should also be a CanExecute function defined. 
And LoginCommand.RaiseCanExecuteChanged called in Login and Password setters